cranelift/codegen/src/isa/aarch64/inst/emit.rs

```diff
@@ -172,6 +172,22 @@ fn enc_conditional_br(
     }
 }
 
+fn enc_test_branch(kind: TestBranchKind, taken: BranchTarget, reg: Reg, bit: u8) -> u32 {
```

Comment:
Similar regarding naming

Reply:
And can you add a test to `emit_tests.rs` for this new instruction format?

Reply:
Would you be ok omitting that? I personally find the `emit_tests.rs` files onerous and unwieldy to update when the golden tests serve basically the same purpose and are much easier to work with.

Reply:
Sure